src/Altinn.App.Core/Models/Layout/GroupComponent.cs

```diff
@@ -0,0 +1,29 @@
+using System.Text.Json;
+
+using Altinn.App.Core.Models.Expressions;
+
+namespace Altinn.App.Core.Models.Layout;
+
+/// <summary>
+/// Tag component to signify that this is a group component
+/// </summary>
+public class GroupComponent : BaseComponent
```

Comment:
Any particular reason why this isn't in the Components namespace with the other components?

Reply:
After trying to explain my reasoning, that seemed to be sound when I put them in separate namespaces, I hope you will accept the answer “no, not anymore”.

Reply:
[ad296dc](https://github.com/Altinn/app-lib-dotnet/pull/23/commits/ad296dc06c4c7617099a33451e282da243177168)